Junk4Dollars-appTests/ShowAuctions/AuctionListViewController.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+import UIKit
+import Junk4Dollars_app
+
+class AuctionListViewControllerTests: XCTestCase {
+    func buildController() -> AuctionListViewController {
+        var controller: AuctionListViewController!
+        controller = AuctionListViewController()
+        let tableView = UITableView()
+        controller.auctionTableView = tableView
+        controller.viewDidLoad()
+        return controller
+    }
+
+    func testControllerSetsDataSource() {
+        let controller = buildController()
+        XCTAssertFalse(controller.auctionTableView.dataSource == nil)
```

Comment:
Is there a way to assert that the datasource is set to the instance variable set in the controller? Many testing frameworks have a way to assert that two values are the same instance.

Reply:
See thread -> https://8thlight.slack.com/archives/GNQJVC52Q/p1573143969004500